src/pages/InstanceTabs/InstanceTabs.tsx

```diff
@@ -86,34 +89,34 @@ const InstanceTabs = () => {
   return (
     <CommandHistoryContextProvider>
       <div
-        className={cn("relative mx-auto flex h-full w-full flex-row justify-center @container", tab.width)}
+        className={cn(
+          'relative mx-auto flex h-full w-full flex-row justify-center @container',
+          tab.width
+        )}
         key={uuid}
       >
-        {InstanceTabListMap[instance.game_type].map(
-          (tab) =>
-            tab.path === path && (
-              <div
-                className="gutter-stable -mx-3 flex grow flex-row items-stretch overflow-y-auto pl-4 pr-2"
-                key={`${instance.name}-${tab.title}`}
-              >
-                <div className="flex h-fit min-h-full w-full flex-col gap-16 pt-6 pb-10 focus:outline-none">
-                  {tab.displayTitle && <div className="flex font-title text-h1 font-bold leading-tight text-gray-300">
-                    {tab.displayTitle}
-                    {tab.title === 'Console' && (
-                      <Label
-                        size="medium"
-                        className="ml-2 mt-[6px]"
-                        color={stateToLabelColor[instance.state]}
-                      >
-                        {instance.state}
-                      </Label>
-                    )}
-                  </div>}
-                  {tab.content}
-                </div>
+        <div
+          className="gutter-stable -mx-3 flex grow flex-row items-stretch overflow-y-auto pl-4 pr-2"
+          key={`${instance.name}-${tab.title}`}
+        >
+          <div className="flex h-fit min-h-full w-full flex-col gap-16 pt-6 pb-10 focus:outline-none">
+            {tab.title && (
```

Comment:
it would be nice if you can merge 0.4.3 into your branch to crush up some merge conflicts. I'm worried about merge conflict with Kevin Wang's changes to tab.title